### ISSUE TITLE
Add Concept to Include Skills Used

### ISSUE THREAD

**Issue by kayaelle** [w3c-ccg/vc-ed-models#23]
In addition to skills achieved, we may wish to add a representation of skills contributed. 

For example: a person may achieve creating a mobile app primarily using skills they already have such as javascript but they may have improved these skills or used these skills as a scaffold for new skills such as adding the mobile app to the apple store. 

In a single course or series/pathway of courses, the scaffolding is often intentional. Outside of a prescribed context, we build skills upon previous ones and when credentials are being evaluated, the consumers of the credential may require clearer articulation of the contextualities and characteristics of the achievement. 

This could be similar to prerequisites but less formal.

**Comment by anthonycamilleri** [comment]
@kayaelle I agree with the need to represent these concepts.Thus, we may want to represent 'inputs' more widely for any concept.

These inputs could be:
- skills brought into a formalised process
- completed educational activities
- possession of certain diplomas.

None of these are 'new claims' but rather are supporting documentation to existing claims, so I would argue, that rather than a new high level concept, these are a characteristic/property of the existing concepts.
